### PR TITLE
Fix copydocs: Move sys path insert up to fix import

### LINKED ISSUE #6790: Investigate `copydocs.py` errors
When following the instructions outlined [here](https://github.com/internetarchive/openlibrary/wiki/Loading-Production-Book-Data), `copydocs.py` fails with a `ModuleNotFoundError`.  It's unclear to me whether the documentation is incorrect, or if `copydocs.py` is broken.

### Steps to Reproduce

1. Go to ...
2. Do ...

* Actual:
* Expected:

### Details

- **Logged in (Y/N)?** N/A
- **Browser type/version?** N/A
- **Operating system?** Ubuntu 20.04
- **Environment (prod/dev/local)?** local

### Stakeholders

@Suhas-13, who identified this issue [here](https://github.com/internetarchive/openlibrary/issues/5810#issuecomment-1196455130).

### PR DESCRIPTION
**Pull request by Suhas-13**
Closes #6790 

Fix #6790 by moving sys.path.insert up to allow import call to work

### Testing

Run copydocs.py on example work

### Stakeholders

@jimchamp

### CODE REVIEW

**Comment by cclauss** [comment]
An alternative approach to this is via the use of [`scripts/_init_path.py`](https://github.com/internetarchive/openlibrary/blob/master/scripts/_init_path.py)

Usage: https://github.com/internetarchive/openlibrary/search?q=_init_path

The nice thing about using `scripts/_init_path.py` is that linters like `flake8` and `isort` accept it.

**Comment by jimchamp** [comment]
Thanks @cclauss!  That worked for me when I tested locally.

File: scripts/copydocs.py
Comment (jimchamp) [inline scripts/copydocs.py:12]:
```suggestion
import _init_path  # noqa: E402,F401
```
With this change, we can also remove `sys.path.insert(0, ".")` (line 11), and the `import scripts._init_path` line (line 14).

**Review by jimchamp — CHANGES_REQUESTED** [review]
Thanks @Suhas-13!  Happy to merge this once @cclauss's suggestion has been implemented.

**Comment by Suhas-13** [comment]
Updated it @jimchamp

File: scripts/copydocs.py
Comment (Suhas-13) [reply scripts/copydocs.py:12]: I made that change, but it seems to cause the python test to fail.

**Comment by jimchamp** [comment]
I'm going to defer to @cclauss and @cdrini for advice about fixing the failing test.

Do either of you know how to resolve the following?
```
ImportError while importing test module '/home/runner/work/openlibrary/openlibrary/scripts/tests/test_copydocs.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
scripts/tests/test_copydocs.py:1: in <module>
    from ..copydocs import copy, KeyVersionPair
scripts/copydocs.py:11: in <module>
    import _init_path
E   ModuleNotFoundError: No module named '_init_path'
```

**Comment by mekarpeles** [comment]
@Suhas-13 can we see if copydocs works with the new revised instructions which include setting the PYTHONPATH? It's possible we can avoid having to add this change, which would be great! (Apologies if we sent you down a rabbit hole)

https://github.com/internetarchive/openlibrary/wiki/Loading-Production-Book-Data

@cdrini maybe it's possible for us to add `PYTHONPATH=.` to our docker-compose setup (at least for dev)

**Comment by jimchamp** [comment]
I was able to import records using the updated `copydocs` instructions.  I'm going to go ahead and close this PR and the linked issue.
@Suhas-13, thanks for creating this PR!  Even though it didn't get merged, your efforts have helped us troubleshoot and solve the issue.

**Comment by Suhas-13** [comment]
Sorry I didn't see this thread. Glad this is resolved.

**Pull request closed without merging**
